### PR TITLE
Allow the user to provide a file path for creating a browser driver

### PR DESCRIPTION
**Pull request by leggebroten**

### CODE REVIEW

**Comment by leggebroten** [comment]
Yes.  I can send an example.  I could update the README.md too to help
clarify the option.

I'm in the process of making another modification to world.js to extract
the driver creation to their own files due to a bug.  (Mismatched
chromedrivers)

**Comment by leggebroten** [comment]
My "selenium-cucumber-js.json" file
{
  "step_def_dir": "./features/step_definitions",
  "default_browser": "./browser_drivers/chrome.js",
  "timeout": 30000
}

My personal chrome driver file.  Basically the same but will use my version of the chromedriver
'use strict';
(function () {
    let selenium = require('selenium-webdriver');
    let chromedriver = require('chromedriver');

    module.exports = function () {
        let driver = new selenium.Builder().withCapabilities({
            browserName: 'chrome',
            javascriptEnabled: true,
            acceptSslCerts: true,
            chromeOptions: {
                args: ['start-maximized']
            },
            path: chromedriver.path
        }).build();

        // driver.manage().window().setSize(1000, 1000);

        return driver;
    }
})();

**Comment by leggebroten** [comment]
Hello?

File: runtime/world.js [hunk 1]
```diff
@@ -60,6 +60,15 @@ function getDriverInstance() {
                 },
                 path: chromedriver.path
             }).build();
+        } break;
+
+        // try to load from file
+        default: {
+            var driverFileName = path.resolve(process.cwd(), browserName);
+            if (!fs.isFileSync(driverFileName)) {
+                throw "Could not find driver file: " + driverFileName;
```
Comment (john-doherty) [inline runtime/world.js]: Should this be an Error instance rather than a plain string?

File: runtime/chromeDriver.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use strict";
+(function(){
```
Comment (john-doherty) [inline runtime/chromeDriver.js]: I'm not sure any of these need an iffy wrapper

**Pull request merged**